src/export/meetbouten/__init__.py

```diff
@@ -0,0 +1,57 @@
+import re
+
+from export.api import API
+from export.meetbouten.config import MeetboutExportConfig, MetingenExportConfig, ReferentiepuntenExportConfig
+from export.meetbouten.types import type_convert
+
+
+CONFIG_MAPPING = {
+    'meetbouten': MeetboutExportConfig,
+    'metingen': MetingenExportConfig,
+    'referentiepunten': ReferentiepuntenExportConfig,
+}
+
+
+def _export_entity(entity, format):
+    """Exports a single entity
+
+    The export file specifications can be found in:
+        https://www.amsterdam.nl/stelselpedia/meetbouten-index/producten/prodspecs-mebo/
+
+    Headers:       None
+    Separator:     |
+    String marker: $$
+
+
+    The export format is a string containing the attributes and types to be converted.
+    A declarative way of describing exports is used:
+    The epxort format is used both to read the attributes and types and to write the correct output format
+
+    :param meetbout:
+    :return:
+    """
+    pattern = re.compile('(\w+):(\w+)\|?')
+    export = format
+    for (attr_name, attr_type) in re.findall(pattern, format):
+        attr_value = type_convert(attr_type, entity.get(attr_name, None))
+        export = export.replace(f'{attr_name}:{attr_type}', attr_value)
+    return export
+
+
+def export_meetbouten(collection, host, file):
+    """Export a collection from catalog meetbouten to a file
```

Comment:
Why is meetbouten still in the name?

Reply:
Discussed to take into account on next catalog export